core/src/main/scala/cats/syntax/functor.scala

```diff
@@ -62,3 +65,20 @@ final class FunctorTuple2Ops[F[_], A, B](private val fab: F[(A, B)]) extends Any
    */
   def unzip(implicit F: Functor[F]): (F[A], F[B]) = F.unzip(fab)
 }
+
+final class IfFOps[F[_]](private val fa: F[Boolean]) extends AnyVal {
+
+  /**
+   * Lifts `if` to Functor
+   *
+   * Example:
+   * {{{
+   * scala> import cats.Functor
+   * scala> import cats.implicits.catsStdInstancesForList
```

Comment:
Side note, I think we may not need the instances import anymore in modern Cats. But we can leave modernization of scaladocs to another issue/PR :)

Reply:
If I understand it correctly, it should be possible simply to remove `import cats.implicits.catsStdInstancesForList`.  `import cats.implicits._` is obsolete starting from v2.2.0. IMO I'd be better not to encourage newcomers to make use of old-fashioned syntax in any way.